runhouse/servers/env_servlet.py

```diff
@@ -183,14 +183,12 @@ async def adelete_local(self, key: Any):
     async def aclear_local(self):
         return await obj_store.aclear_local()
 
-    def _get_env_cpu_usage(self):
+    def _get_env_cpu_usage(self, cluster_config: dict = obj_store.cluster_config):
```

Comment:
setting the default to a mutable property like that feels wrong, can we just set to `None` instead and then do:
`cluster_config = cluster_config or obj_store.cluster_config`?